build-logic/src/main/kotlin/Common.kt

```diff
@@ -5,7 +5,7 @@ import org.jetbrains.kotlin.gradle.targets.jvm.tasks.KotlinJvmTest
 fun Project.commonSetup() {
   pluginManager.withPlugin("org.jetbrains.kotlin.jvm") {
     tasks.register("ft") {
-      if (this@commonSetup.name != "apollo-gradle-plugin") {
+      if (this@commonSetup.name !in setOf("apollo-gradle-plugin", "intellij-plugin")) {
```

Comment:
Removed intellij-plugin from `ft` because they're a tiny bit slow but also very prone to flakes. I've had them fail on multiple occasions.